MintegralAdapter/src/main/java/com/chartboost/mediation/mintegraladapter/MintegralAdapter.kt

```diff
@@ -119,36 +117,36 @@ class MintegralAdapter : PartnerAdapter {
         PartnerLogController.log(SETUP_STARTED)
 
         val appId = Json.decodeFromJsonElement<String>(
-            (partnerConfiguration.credentials as JsonObject).getValue(APP_ID_KEY)
+            partnerConfiguration.credentials.getValue(APP_ID_KEY)
         ).trim()
         val appKey = Json.decodeFromJsonElement<String>(
-            (partnerConfiguration.credentials as JsonObject).getValue(APP_KEY_KEY)
+            partnerConfiguration.credentials.getValue(APP_KEY_KEY)
```

Comment:
And this is working fine for you?

I know the IDE says that's useless code, but I was getting crashes before adding it. Something may have changed since then though

Reply:
Yeah it worked for me just earlier.